coalib/bearlib/__init__.py

```diff
@@ -53,6 +53,9 @@ def deprecate_settings(**depr_args):
     WARNING:root:The setting `old` is deprecated. Please use `new` instead.
     WARNING:root:The value of `old` and `new` are conflicting. `new` will...
     coala is always written with lowercase `c`.
+    >>> @deprecate_settings(new='old')
```

Comment:
Need to rededine just for the test else arg caching will kick in and the warning won't be printed :/